src-docs/src/views/steps/steps_example.js

```diff
@@ -17,22 +17,32 @@ import { EuiStepHorizontal } from '../../../../src/components/steps/step_horizon
 import Steps from './steps';
 const stepsSource = require('!!raw-loader!./steps');
 const stepsHtml = renderToHtml(Steps);
+const stepsSnippet = `<EuiSteps steps={steps} />
+`;
 
 import StepsComplex from './steps_complex';
 const stepsComplexSource = require('!!raw-loader!./steps_complex');
 const stepsComplexHtml = renderToHtml(StepsComplex);
+const stepsComplexSnippet = `<EuiSteps steps={complexSteps} />
+`;
 
 import HeadingElementSteps from './heading_element_steps';
 const headingElementStepsSource = require('!!raw-loader!./heading_element_steps');
 const headingElementStepsHtml = renderToHtml(HeadingElementSteps);
+const headingElementStepsSnippet = `<EuiSteps steps={steps} headingElement="h2" />
+`;
 
 import StepsHorizontal from './steps_horizontal';
 const stepsHorizontalSource = require('!!raw-loader!./steps_horizontal');
 const stepsHorizontalHtml = renderToHtml(StepsHorizontal);
+const stepsHorizontalSnippet = `<EuiStepsHorizontal steps={horizontalSteps} />
+`;
 
 import Status from './status';
 const statusSource = require('!!raw-loader!./status');
 const statusHtml = renderToHtml(Steps);
+const statusSnippet = `<EuiSteps steps={stepsWithStatus} />
```

Comment:
This one requires the steps object to be explained 
```suggestion
const statusSnippet = `<EuiSteps 
  steps={{
    title: 'Warning',
    children: 'Example of a warning',
    status: 'warning',
  }}
/>`,
```